lib/pages/DesktopFarm.dart

```diff
@@ -159,7 +155,7 @@ class _DesktopFarmState extends State<DesktopFarm> {
                     if (!isAllFarms) {
                       myController.clear();
                       setState(() {
-                        allFarmsListSearchFilter = allFarmsList;
+                        // allFarmsListSearchFilter = allFarmsList;
```

Comment:
if this isnt needed any more just remove it

Reply:
I removed it